server/src/test/java/io/unitycatalog/server/sdk/access/SdkStagingTableAccessControlCRUDTest.java

```diff
@@ -0,0 +1,149 @@
+package io.unitycatalog.server.sdk.access;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.api.TablesApi;
+import io.unitycatalog.client.api.TemporaryCredentialsApi;
+import io.unitycatalog.client.model.ColumnInfo;
+import io.unitycatalog.client.model.ColumnTypeName;
+import io.unitycatalog.client.model.CreateStagingTable;
+import io.unitycatalog.client.model.CreateTable;
+import io.unitycatalog.client.model.DataSourceFormat;
+import io.unitycatalog.client.model.GenerateTemporaryTableCredential;
+import io.unitycatalog.client.model.SecurableType;
+import io.unitycatalog.client.model.StagingTableInfo;
+import io.unitycatalog.client.model.TableInfo;
+import io.unitycatalog.client.model.TableOperation;
+import io.unitycatalog.client.model.TableType;
+import io.unitycatalog.client.model.TemporaryCredentials;
+import io.unitycatalog.server.base.ServerConfig;
+import io.unitycatalog.server.exception.ErrorCode;
+import io.unitycatalog.server.persist.model.Privileges;
+import io.unitycatalog.server.utils.TestUtils;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Access control tests for staging tables that require authorization to be enabled.
+ *
+ * <p>This test class extends {@link SdkAccessControlBaseCRUDTest} which provides common support
+ * functionality for SDK-based access control tests, including user management, permission
+ * management, and resource setup.
+ */
+public class SdkStagingTableAccessControlCRUDTest extends SdkAccessControlBaseCRUDTest {
```

Comment:
technically this test is not testing CRUD (create, read, update, delete) as staging table cannot be updated or deleted.